spt.vcxproj

```diff
@@ -255,18 +260,28 @@
   </PropertyGroup>
   <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">
     <PreBuildEvent>
-      <Command />
+      <Command>IF NOT EXIST SPTLib\.git (
+    SET SUBMODULE=SPTLib
+    git submodule update --init -j %NUMBER_OF_PROCESSORS% %SUBMODULE% &amp;&amp;^
+    echo Successfully checked out %SUBMODULE%.                                                       &amp;&amp;^
+    echo Please reload all projects.
+    EXIT /b 1
+)
+IF NOT EXIST SDK\orangebox\.git (
+    SET SUBMODULE=SDK\orangebox
+    git submodule update --init -j %NUMBER_OF_PROCESSORS% %SUBMODULE% &amp;&amp;^
+    echo Successfully checked out %SUBMODULE%.
+)</Command>
```

Comment:
I'm not sure I like checking out submodules from a pre-build event. I certainly wouldn't expect something like this in a pre-build event (although I don't deal with MSVC projects all that much, maybe it's actually common?). I'd rather leave it as-is.